### PR TITLE
fix(18859): use label property for MCDA axis if it's provided. Remove duplicated bivariate statistics type definitions

### PR DESCRIPTION
**Pull request by albaranau**
https://kontur.fibery.io/Tasks/Task/MCDA-axis-ignores-label-property-18859

## Summary by CodeRabbit

- **Bug Fixes**
  - Corrected a typo in an import statement for `axisDTOtoAxis` to ensure functionality.

- **Refactor**
  - Updated import paths for `BivariateStatisticsResponse` and `StatDTO` to new project structure locations.
  - Modified `axisDTOtoAxis` function to accept updated `AxisDTO` type, enhancing label assignment logic.
  - Made `label` field optional in `AxisDTO` type for increased data structure flexibility.

### CODE REVIEW

File: src/utils/bivariate/helpers/converters/axixDTOtoAxis.ts
Comment (vkozio) [inline src/utils/bivariate/helpers/converters/axixDTOtoAxis.ts:1]: typo in filename

**Review by vkozio — CHANGES_REQUESTED** [review]
please move DTO types used in api into relevant core/api/... files

File: src/core/resources/bivariateStatisticsResource/types.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import type {
   ColorCombination,
 } from '~utils/bivariate';
 
-type AxisDTO = Omit<Axis, 'id' | 'label'>;
+export type AxisDTO = Omit<Axis, 'id' | 'label'> & { label?: string | null };
```
Comment (Natallia-Harshunova) [inline src/core/resources/bivariateStatisticsResource/types.ts]: Does `label` property can be both `null` and `undefined`?

File: src/utils/bivariate/helpers/converters/axixDTOtoAxis.ts
Comment (albaranau) [reply src/utils/bivariate/helpers/converters/axixDTOtoAxis.ts:1]: fixed

File: src/core/resources/bivariateStatisticsResource/types.ts (hunk 1 above)
Comment (albaranau) [reply src/core/resources/bivariateStatisticsResource/types.ts]: You are right, this property doesn't need to be optional. Fixed to string | null

**Comment by albaranau** [comment]
> please move DTO types used in api into relevant core/api/... files

Regardling AxisDTO - it's currently placed in the folder with the atom (`bivariateStatisticsResourceAtom`) that uses it (the atom pulls data with `apiClient` directly, without using helper api functions.

@vkozio  Do we want to extract that api call into separate function? Considering there's quite a bit of error handling inside, that should be a separate task, probably

**Review by Natallia-Harshunova — APPROVED** [review]

**Review by vkozio — APPROVED** [review]

**Pull request merged**